src/views/dashboard/index.vue

```diff
@@ -154,9 +162,9 @@ export default {
       this.$root.showError('Could not fetch user', err)
     })
 
-    this.axios.get(this.services['oms-events'] + '/mine/participating').then((response) => {
-      this.axios.get(this.services['oms-statutory'] + '/events/mine').then((response2) => {
-        const input = response.data.data.concat(response2.data.data)
+    this.axios.get(this.services['oms-events'] + '/mine/participating').then((eventResponse) => {
```

Comment:
one thing, it's `eventsResponse` since it's `oms-events` queried and not `oms-event` :D